### PR TITLE
Add testcase for file zstream tracker

### PR DESCRIPTION
**Pull request by huiwangredhat**
Fixes OSIDB-1967

### CODE REVIEW

File: features/pages/flaw_detail_page.py [hunk 1]
```diff
@@ -539,23 +540,26 @@ def set_new_affect_inputs(self, external_system = 'jira', affectedness_value = '
         self.click_btn('saveBtn')
         self.wait_msg('affectCreatedMsg')
         self.wait_msg('flawSavedMsg')
-        time.sleep(2)
+        time.sleep(60)
```
[the remark this answers is not in the document]
Comment (huiwangredhat) [reply features/pages/flaw_detail_page.py:543]: I found there always added more than one affect even though I deleted the affect before each case. I guess it hasn't deleted the affect or it hasn't refreshed before adding a new affect. I am still debugging the issue. Pls help me to find out the root cause if you are interested in it.

[the remark this answers is not in the document]
Comment (RichardXuan) [reply features/pages/flaw_detail_page.py:543]: Maybe count the affect number, and wait count times `affectCreatedMsg ` popup message. What do you think.

**Review by RichardXuan — APPROVED** [review]
Others LGTM

File: features/pages/flaw_detail_page.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (huiwangredhat) [reply features/pages/flaw_detail_page.py]: I deleted the sleep time. The rootcause is bug OSIDB-3132.

**Comment by huiwangredhat** [comment]
Close OSIDB-1967
Close OSIDB-1968

**Pull request merged**